cmsController.php

```diff
@@ -0,0 +1,195 @@
+<?php
+$db = new PDO('mysql:host=127.0.0.1; dbname=portfolioKyam', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+/**
+ * Gets about sections name, content from Db and turns into an array
+ *
+ * @param $db PDO to select from
+ * @return assoc array of about sections and content
+ */
+function FillAbout (PDO $db):array {
+    $query=$db->prepare("SELECT `name`,`content` FROM `staticContent` WHERE `name` != 'submitAbout' ORDER BY `name` DESC;");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**
+ * Gets portfolio items names from DB and turns into an array
+ *
+ * @param $db PDO to select from
+ * @return assoc array of pf item names
+ */
+function portfolioList (PDO $db):array {
+    $query=$db->prepare("SELECT `title` FROM `portfolioItems` WHERE `deleted`!=1;");
+    $query->execute();
+    return $query->fetchall();
+}
+
+/**
+ * Takes an array of $items and turns into a string to make a drop-down list
+ *
+ * @param $items array of items (options) to be made into a drop-down
+ * @return string of options
+ */
+function makeDropDown (array $items): string {
```

Comment:
this function is identical to imgdropdown except u have the same value and display, u could just have 1 function instead of 2, call it, createDropDown

Reply:
resolved

Reply:
but it broke everything...

Reply:
everything fixed.